src/java/org/apache/cassandra/cql3/statements/TransactionStatement.java

```diff
@@ -19,42 +19,28 @@
 package org.apache.cassandra.cql3.statements;
 
 import java.nio.ByteBuffer;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-import java.util.SortedSet;
-import java.util.TreeSet;
+import java.util.*;
```

Comment:
nit: Restore single imports?

Reply:
Ran optimize imports with the right style (hopefully for real this time!)